ts/components/conversation/AddMentions.tsx

```diff
@@ -71,8 +75,8 @@ class Mention extends React.Component<MentionProps, MentionState> {
     clearInterval(this.intervalHandle);
   }
 
-  private tryRenameMention() {
-    const found = this.findMember(this.props.text.slice(1));
+  private async tryRenameMention() {
+    const found = await this.findMember(this.props.text.slice(1));
     if (found) {
       this.setState({ found });
```

Comment:
maybe set the state regardless?

Reply:
I guess we could do that, but I don't see any real benefit (it won't simplify code much and we will just end up re-setting the same value in that case.